site/layouts/partials/content/_index.html

```diff
@@ -1,6 +1,6 @@
 <!-- Content -->
 {{ if or (.Content) (.Params.image) }}
-<div class="content">
+<div class="content has-text-centered">
```

Comment:
There should be a div just around the image, right now it's applying to all the content. 😬